corehq/apps/reports/standard/cases/filters.py

```diff
@@ -149,6 +164,43 @@ def get_value(cls, request, domain):
         return ret
 
 
+class SensitiveCaseProperties(CaseListExplorerColumns):
+    slug = "sensitive_properties"
+    label = gettext_lazy("De-identify options")
+    template = "reports/filters/sensitive_columns.html"
+    EXCLUDE_PROPERTIES = [
+        '@case_id', '@case_type', '@owner_id', '@status', 'closed_on', 'last_modified', 'date_opened',
+        'commcare_case_copy'
+    ]
+
+    @property
+    def filter_context(self):
+        context = super(SensitiveCaseProperties, self).filter_context
+        initial_values = self.get_value(self.request, self.domain)
+
+        context.update({
+            'initial_value': json.dumps(initial_values),
+            'column_suggestions': json.dumps(self.get_column_suggestions()),
+            'property_label_options': self.property_label_options
+        })
+        return context
+
+    @property
+    def property_label_options(self):
+        return [
+            {'type': DEID_ID_TRANSFORM, 'name': gettext_lazy('Sensitive ID')},
+            {'type': DEID_DATE_TRANSFORM, 'name': gettext_lazy('Sensitive Date')},
+        ]
+
+    def get_column_suggestions(self):
+        case_properties = get_flattened_case_properties(self.domain, include_parent_properties=False)
+        special_properties = [
+            {'name': prop, 'case_type': None, 'meta_type': 'info'}
+            for prop in SPECIAL_CASE_PROPERTIES if prop not in self.EXCLUDE_PROPERTIES
+        ]
```

Comment:
It looks like `self.EXCLUDE_PROPERTIES` is almost all of `SPECIAL_CASE_PROPERTIES`. You could simplify this, and make what's happening here more obvious, if you just list the remaining included properties:
```suggestion
        special_properties = [
            {'name': prop, 'case_type': None, 'meta_type': 'info'}
            for prop in ('name', 'case_name', 'external_id')
        ]
```